spec/jobs/cancel_outdated_reservations_job_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CancelOutdatedReservationsJob, type: :job do
+  describe '#perform_later' do
+    it 'enqueues job' do
+      ActiveJob::Base.queue_adapter = :test
+      expect { described_class.perform_later }.to have_enqueued_job
+    end
+  end
+
+  describe '#perform_now' do
+    let(:reservation) { create(:reservation) }
```

Comment:
I know you have a default value for a `status` column but I can't see it looking at the factory for this variable so for a better readability or changes in a future you could define status like this.
```suggestion
    let(:reservation) { create(:reservation, status: Reservation::CREATED) }
```

Reply:
You could even create `trait` that would tell more what reservation you are creating (because when status is created you would like to have other columns with a specific data)
```ruby
FactoryBot.define do
  factory :reservation do
  trait :created do
    status { Reservation::CREATED }
  end
end
```